src/cairoUtilFuncGen/storage/dynArrayPushWithArg.ts

```diff
@@ -157,7 +153,8 @@ export class DynArrayPushWithArgGen extends StringIndexedFuncGen {
     return {
       name: funcName,
       code: [
-        `func ${funcName}${implicits}(loc: felt, value: ${inputType}) -> (){`,
+        `#[implicit(warp_memory)]`,
```

Comment:
The function had warp memory implicit added conditionally. Imo we should keep it that way. Wyt @rodrigo-pino?